web/satellite/src/api/projectMembers.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+import apollo from '@/utils/apolloManager';
+import gql from "graphql-tag";
+
+// Performs graqhQL request.
+// Throws an exception if error occurs
+export async function addProjectMember(userID: string, projectID: string): Promise<any> {
+    let response = await apollo.mutate(
+        {
+            mutation: gql(`
+                mutation {
+                    addProjectMember(
+                        projectID: "${projectID}",
+                        userID: "${userID}"
+                    ) {id}
```

Comment:
I thin that you should request TeamMember info. We don't need TeamMember ID on frontend